ebuild-writing/common-mistakes/text.xml

```diff
@@ -404,8 +404,9 @@ the guidelines to use TABS rather than spaces. So <e>please</e> use tabs!
 <body>
 
 <p>
-I'm often guilty of this. Remember to run repoman over your ebuilds so it can
-tell you if there is trailing whitespace at the end of lines or on empty lines.
+I'm often guilty of this. Remember to run <c>repoman</c> or <c>pkgcheck</c>
```

Comment:
"Third person form should be used rather than first."
https://devmanual.gentoo.org/appendices/contributing/index.html#style-guidelines

Reply:
That was there before, but sure.

Reply:
The chapter is about common mistakes, and the first sentence just says that it's a common mistake. :smile:

So:
```suggestion
Remember to run <c>repoman</c> or <c>pkgcheck</c>
```

Reply:
Oh, very good!